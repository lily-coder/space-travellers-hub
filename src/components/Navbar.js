/* eslint no-unused-vars : "off" */
import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../logo.png';

const Navbar = () => {
  const r = '';
  return (
    <nav className='header-navbar'>
      <div className='logo'>
        <img
          alt=""
          src={logo}
          width="35"
          height="35"
          className="header-logo"
        />
        {''}
        Space Traveler's Hub
      </div>
      <div className="header-nav">
        <ul className="header-nav_ul">
          <li className="rockets">
            <NavLink to='/' exact>Rockets</NavLink>
          </li>
          <li className="missions">
          <NavLink to='/Missions' exact>Missions</NavLink>
          </li>
          <li className="profile">
          <NavLink to='/Profile' exact>My Profile</NavLink>
          </li>
        </ul>
      </div>
    </nav>
  );
};

export default Navbar;