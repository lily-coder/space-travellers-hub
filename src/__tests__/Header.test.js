import TestRenderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect.js';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configStore.js';
import Navbar from '../components/Navbar.js';

describe('Render the header', () => {
  it('Renders the header', () => {
    const header = TestRenderer.create(
      <Provider store={store}>
        <Router>
          <Navbar />
        </Router>
      </Provider>,
    ).toJSON();
    expect(header).toMatchSnapshot();
  });
});
