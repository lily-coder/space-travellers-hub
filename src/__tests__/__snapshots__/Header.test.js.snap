// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the header Renders the header 1`] = `
<header
  className="header-section"
>
  <nav
    className="header-navbar"
  >
    <div
      className="logo"
    >
      <img
        alt=""
        className="header-logo"
        height="50"
        src="logo.png"
        width="50"
      />
      
      <p
        className="space-travellers"
      >
        Space Traveler's Hub
      </p>
    </div>
    <div
      className="header-nav"
    >
      <ul
        className="header-nav_ul"
      >
        <li
          className="rockets"
        >
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Rockets
          </a>
        </li>
        <li
          className="missions"
        >
          <a
            aria-current={null}
            href="/Missions"
            onClick={[Function]}
          >
            Missions
          </a>
        </li>
        <li
          className="profile"
        >
          <a
            aria-current={null}
            href="/Profile"
            onClick={[Function]}
          >
            My Profile
          </a>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
